### DISCUSSION TITLE
Possible to pass variables in Exec template function

### DISCUSSION THREAD

**Discussion by geekyfox90** [helmfile/helmfile#349] [Ideas]
Hello guys,

I would like to pass variavles into the Exec template function, I don't know if that's possible, so I want to share with you.

I would to acheive the following:

```
mysetting: |
{{ exec "./mycmd" (list "{{.Values.Arg1}}" "{.Values.Arg2}}" "--flag1") | indent 2 }}
```

Where environments/default/values.yaml looks like this: 

```
---
Arg1: "test"
Arg2: "test"
```

**Comment by yxxhero** [comment]
@geekyfox90 any thing I can help you?

**Comment by geekyfox90** [comment]
@yxxhero I want to achieve what's in the example I mentioned to be able to pass variables to Exec template function, but it doesn't seem to work

**Comment by yxxhero** [comment]
{{ exec "./mycmd" (list "{{.Values.Arg1}}" "{{.Values.Arg2}}" "--flag1") | indent 2 }}

**Reply by yxxhero** [reply]
or maybe 
{{ $a := .Values.Arg1 }}
{{ $b := .Values.Arg2 }}
{{ exec "./mycmd" (list $a $b "--flag1") | indent 2 }}
